sites/programetv.ro/programetv.ro.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     }
     const day = date.day()
 
-    return `https://www.programetv.ro/post/${channel.site_id}/${daysOfWeek[day]}/`
+    return `https://www.programetv.ro/program-tv/${channel.site_id}/${daysOfWeek[day]}/`

```

Comment:
You must also update the url used in test.

Reply:
updated url in test as well.